CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ All notable changes to this project will be documented in this file.
 - Update the JSON schema with the required fields for the output content of the migration tool ([#4375](https://github.com/wazuh/wazuh-qa/pull/4375)) \- (Tests)
 - Update framework known flaws file ([#4443](https://github.com/wazuh/wazuh-qa/pull/4443)) \- (Tests)
 - Align migration tool system tests to the tool's new output directory structure ([#4561](https://github.com/wazuh/wazuh-qa/pull/4561)) \- (Tests)
+- Updated Filebeat module to 0.3 ([4700](https://github.com/wazuh/wazuh-qa/pull/4700)) \- (Framework)
```

Comment:
```suggestion
- Updated Filebeat module to 0.3 ([#4700](https://github.com/wazuh/wazuh-qa/pull/4700)) \- (Framework)
```

In addition, we usually update the changelog upwards.

Reply:
Done in 90611ad117389b2c1d1f9f657ee710a445adc1f8